### PR TITLE
Make route services pie chart responsive

### LINKED ISSUE #1274: Long route names shrink pie chart for routes with alt services
Normal:

openshift_web_console

Longer names:

openshift_web_console

### PR DESCRIPTION
**Pull request by spadgett**
Fix problems on mobile and with long names.

Fixes #1274

### CODE REVIEW

**Comment by spadgett** [comment]
Mobile:

screen shot 2017-02-20 at 1 57 02 pm

Desktop:

screen shot 2017-02-20 at 1 57 14 pm

File: app/scripts/directives/routeServicePie.js [hunk 1]
```diff
@@ -69,14 +69,17 @@ angular.module('openshiftConsole')
...
         function updateChart() {
           var data = {
-            columns: []
+            columns: [],
+            names: {}
           };
 
           if ($scope.route) {
             data.columns.push(getData($scope.route.spec.to));
+            data.names[$scope.route.spec.to.name] = _.trunc($scope.route.spec.to.name, { length: 30 });
```
Comment (jwforres) [inline app/scripts/directives/routeServicePie.js]: so the full name isn't available on hover either?

Comment (spadgett) [inline app/scripts/directives/routeServicePie.js]: Updated with a fix

**Comment by jwforres** [comment]
[merge]

**Pull request merged**